src/inner-slider.js

```diff
@@ -188,6 +195,13 @@ export class InnerSlider extends React.Component {
     //   this.props.onLazyLoad([leftMostSlide])
     // }
     this.adaptHeight();
+
+    if (prevProps.touchMove !== this.props.touchMove) {
+      const method = touchMove ? "add" : "remove";
```

Comment:
is touchMove defined? Should be this.props.touchMove?

Reply:
Sorry, fixed!